docs/core/settings/types.md

```diff
@@ -186,6 +186,10 @@ setting by using `inline:` prefix in the value. For example userdb could be
 returning `crypt_global_public_key_file=inline:<public key content>` without
 having to use files.
 
+If you use `%{variables}` in file names, the expansion is delayed until the
```

Comment:
Two paragraphs up it still says `The file paths do not support [[link,settings_variables,%variables]].`

Reply:
Perhaps keep it that way? `If you use [[link,settings_variables,%variables]] in file names`

Reply:
These weren't changed (the two comments are two issues).